components/api/resource/cellery-hub-api.yaml

```diff
@@ -202,7 +232,7 @@ definitions:
   OrgListResponse:
     type: object
     properties:
-      total:
+      totalCount:
```

Comment:
This is now count right ? Could you fix in all the places if we are maintaining the swagger ?